suite-native/module-accounts-import/src/components/XpubHintModal.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { ReactNode, useState } from 'react';
+import { TouchableOpacity, View } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { BottomSheet, Box, Button, Text, VStack } from '@suite-native/atoms';
+import { Icon } from '@trezor/icons';
+import { NetworkType } from '@trezor/connect/lib/constants/network';
+import { Video } from '@suite-native/video-assets';
+
+type XpubScanHintSheet = {
+    networkType: NetworkType;
+};
+
+const sheetTriggerStyle = prepareNativeStyle(_ => ({
+    flexDirection: 'row',
+    width: '100%',
+    justifyContent: 'center',
+    alignItems: 'center',
+}));
+
+const networkToTitleMap = {
+    bitcoin: 'Where is my public key (XPUB)?',
+    ethereum: 'Where is my receive address?',
+    ripple: 'Where is my receive address?',
+} as const as Record<NetworkType, string>;
+
+const EmphasizedText = ({ children }: { children: ReactNode }) => (
+    <Text color="gray900">{children}</Text>
+);
+
+const networkToTextMap = {
+    bitcoin: (
+        <>
+            To view the Bitcoin XPUB of your account, open the Trezor Suite desktop app, plug in
+            your Trezor device, select <EmphasizedText>Accounts</EmphasizedText>, then choose{' '}
+            <EmphasizedText>Show public key</EmphasizedText>.
+        </>
+    ),
+    ethereum: (
+        <>
+            To view the Ethereum and Ethereum Classic receive address of your account, open the
+            Trezor Suite desktop app, plug in your Trezor device, select{' '}
+            <EmphasizedText>Accounts</EmphasizedText>, choose Receive, and click on{' '}
+            <EmphasizedText>Show full address</EmphasizedText>.
+        </>
+    ),
+    ripple: (
+        <>
+            To view the Ripple receive address of your account, open the Trezor Suite desktop app,
+            plug in your Trezor device, select <EmphasizedText>Accounts</EmphasizedText>, choose
+            <EmphasizedText>Receive</EmphasizedText>, and click on{' '}
+            <EmphasizedText>Show full address</EmphasizedText>
+        </>
+    ),
+} as const as Record<NetworkType, ReactNode>;
+
+export const XpubHintModal = ({ networkType }: XpubScanHintSheet) => {
+    const [isVisible, setIsVisible] = useState(false);
+    const { applyStyle } = useNativeStyles();
+    const sheetTitle = networkToTitleMap[networkType] ?? networkToTitleMap.bitcoin;
+    const sheetDescription = networkToTextMap[networkType] ?? networkToTextMap.bitcoin;
+
+    const handleClose = () => setIsVisible(false);
+
+    return (
+        <View>
+            <TouchableOpacity
+                onPress={() => setIsVisible(true)}
+                style={applyStyle(sheetTriggerStyle)}
+            >
+                <Icon name="question" size="medium" color="forest" />
+                <Text color="forest">{`  ${sheetTitle}`}</Text>
```

Comment:
Why the empty spaces? Did you just want to add margin left? If yes, just do so.

Reply:
fixed 95199a91a